src/webfiles/portal/js/juxt.js

```diff
@@ -36,13 +37,49 @@ function initYeah() {
             parent.classList.remove('yeah');
             if(count) count.innerText -= 1;
             wiiuSound.playSoundByName('SE_OLV_MII_CANCEL', 1);
+            if (document.querySelector('#post-' + id + ' .yeah-list')) {
+                var yeahList = document.querySelector('#post-' + id + ' .yeah-list');
+                var matchesMiiSrc = document.querySelector('#nav-menu-me .mii-icon img').src;
 
+                var childElements = yeahList.children;
+                if (childElements != 1){
+                for (var i = 0; i < childElements.length; i++) {
+                  var childImgSrc = childElements[i].querySelector('img').src;
+                  if (childImgSrc === matchesMiiSrc) {
+                    yeahList.removeChild(childElements[i]);
+                    break;
+                  }
+                }
+              }
+            }
```

Comment:
It looks like this uses 2 space indentation. We use 4 here (though eventually I'm moving everything to tabs, but it should be 4 for now)